### PR TITLE
Updated Readme.md with latest screenshots.

### PR DESCRIPTION
**Pull request by ishita1818**
@madhavrathi  review it please.

### CODE REVIEW

**Comment by Logan1x** [comment]
Everything looks good but you need to squash your commits.

**Review by Logan1x — APPROVED** [review]

**Comment by ishita1818** [comment]
Okay!

**Comment by ishita1818** [comment]
@Logan1x Now is that okay ?
Wait lemme resolve these conflicts first.

File: app/build.gradle [hunk 1]
```diff
@@ -20,8 +21,14 @@ android {
 }
 
 dependencies {
-    compile fileTree(dir: 'libs', include: ['*.jar'])
+    implementation 'com.android.support:design:26.1.0'
```
Comment (Logan1x) [inline app/build.gradle]: was this file supposed to come in this PR.

Comment (Logan1x) [reply app/build.gradle]: @ishita1818

**Comment by Logan1x** [comment]
@ishita1818  there is bit problem I am seeing files which were not supposed in this pr.

**Comment by Logan1x** [comment]
> @ishita1818 there is bit problem I am seeing files which were not supposed in this pr.

problem resolved but still seeing conflicts

**Comment by ishita1818** [comment]
I guess now everything is fine.

**Comment by Logan1x** [comment]
Everything is fine now merging it.
CC: @madhavrathi

**Comment by Logan1x** [comment]
Feedback: next time when you make changes create a different branch.
CC: @ishita1818

**Comment by ishita1818** [comment]
Okay @Logan1x .
Thanks for the suggestion.

**Comment by Logan1x** [comment]
@ishita1818  your PR was fix for #16  right?

**Comment by ishita1818** [comment]
Actually that issue was created just for that time when I made changes for saving and sharing image. But now when there are a hell lot of changes after that, @madhavrathi asked me to update the readme in #22 . So, I created this pr.

**Pull request merged**